docs/debugging_tips.md

```diff
@@ -35,19 +35,135 @@ configuration, diagnostic, and troubleshooting data to be provided to Red Hat Te
+* MicroShift RPMs being installed.
+* MicroShift systemd service being present, whether running or not.
+* MicroShift ovnk pods running.
+
+For more information on the plugin design and output format, please check the [enhancement proposal](https://github.com/openshift/enhancements/blob/master/enhancements/microshift/microshift-supportability-tools.md).
 
+The plugin is also available through the usual options to enable/disable them individually or by profile.
 ```bash
-sudo sos report --batch --clean
+$ sos report --list-profiles
+
+sosreport (version 4.3)
+
+The following profiles are available:
+
+ boot            boot, devices, dracut, grub2, services, systemd, udev
+ cluster         conntrack
+ container       cgroups, containers_common, crio, podman, selinux
+ debug           hts, kdump
+ desktop         soundcard, x11
+ hardware        ata, block, crypto, devices, hardware, kernel, logs,
+                 memory, multipath, networking, networkmanager, ntb,
+                 openhpi, pci, processor, scsi, soundcard, teamd, udev,
+                 udisks, usb, x11
+ identity        krb5, ldap, login, nis, pam, ssh, sssd, sudo
+ java            jars
+ kernel          ebpf, kernel, system
+ memory          memory, processor
+ microshift      microshift, microshift_ovn
+ network         conntrack, ebpf, firewall_tables, firewalld,
+                 networking, networkmanager, nss, openssl, teamd,
+                 unbound
+ openshift       selinux
+ packagemanager  dnf, rpm, subscription_manager, yum
+ performance     tuned
+ perl            perl
+ sap             vhostmd
+ security        auditd, nss, openssl, pam, selinux, ssh, sssd
+ services        chrony, nis, ssh, sssd, systemd, sysvipc, unbound
+ storage         ata, block, devicemapper, devices, filesys, iscsi,
+                 logs, lvm2, md, multipath, scsi, xfs
+ sysmgmt         dnf, insights, ldap, subscription_manager, yum
+ system          anaconda, anacron, auditd, boot, cgroups, chrony,
+                 console, cron, crypto, dbus, devices, dnf, ebpf,
+                 firewall_tables, hardware, host, i18n, insights, kdump,
+                 kernel, krb5, kvm, ldap, libraries, libvirt, login,
+                 logrotate, logs, memory, multipath, networking,
+                 networkmanager, openhpi, pam, pci, process, processor,
+                 python, release, rpm, selinux, services, ssh,
+                 subscription_manager, sudo, system, systemd, sysvipc
+                 tuned, udev, udisks, unbound, vhostmd, yum
+ virt            kvm, libvirt, vhostmd, xen
+ webserver       perl
+
+ 24 profiles, 91 plugins
 ```
 
-The report archives can be found in the `/var/tmp/sosreport-*` files.
+Log into the host running MicroShift and execute the following command to generate an obfuscated
+report that should not contain sensitive information (in this example only microshift plugin was
+enabled. Your system's output may vary):
```

Comment:
Do you want to update the command below to explicitly run microshift plugin only? This way, we should not need to explain about different output, etc.

Reply:
Done.